src/Components/Notifications/NotificationItem.tsx

```diff
@@ -216,7 +220,23 @@ NotificationItemLink.defaultProps = {
  * Until we support all notification types in the new activity panel,
  * we only link to the notification detail page for the supported types.
  */
-const getNotificationUrl = (notification: NotificationItem_item$data) => {
+const getNotificationUrl = (
+  notification: NotificationItem_item$data,
+  mode: NotificationListMode = "page"
```

Comment:
I wonder if we should change this parameter to a boolean (e.g. `alwaysOpenAsNotification`) to decouple this function from the concept of Notification mode.

Reply:
This would also prevent us from "re-setting" the mode to "dropdown" on smaller screen sizes.

Reply:
I tried to find a better name for this option, but I actually find mode (dropdown | page) more descriptive, because behaviour actually changes based on how the list of notifications is displayed.